fossunited/fossunited/doctype/grants_funding_directory/grants_funding_directory.py

```
@@ -0,0 +1,259 @@
+            )
+
+            validation_result = validation_response.json()
+
+            # Check if validation failed
+            if validation_response.status_code != 200:
+                error_msg = validation_result.get("message", "Invalid Funding JSON")
+                frappe.throw(f"Validation failed: {error_msg}")
+
+            return validation_result.get("data", json.loads(json_content))
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)